.gitignore

```diff
@@ -5,5 +5,5 @@
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-# Forc output
-**/out/
\ No newline at end of file
+# Forc lock
+Forc.lock
```

Comment:
Maybe it's worth including the `Forc.lock` files, purely to keep track of which commits were used to produce the ABI and binary?

Reply:
Yeah, that's a good idea!